### PR TITLE
Auto-spawn classes for matlab

### PR DESCRIPTION
**Pull request by mingguichen**

### CODE REVIEW

File: matlab/schemas/+map/ScanCoordinates2.m [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+x_offset                    : double                        # x center coordinate in pixels
+y_offset                    : double                        # y center coordinate in pixels
+depth                       : double                        # depth of slice from surface in microns
+tform                       : mediumblob                    # transformation matrix for rotation,scale,flip relative to vessel map
+pxpitch                     : double                        # estimated pixel pitch of the reference map (px)
+ref_key                     : mediumblob                    # key of the reference vessel map
+%}
+
+
+classdef ScanCoordinates2 < dj.Imported
```
Comment (ecobost) [inline matlab/schemas/+map/ScanCoordinates2.m]: Are ScanCoordinates and ScanCoordinates2 needed?

File: matlab/schemas/+shared/SliceTest.m [hunk 2]
```diff
@@ -0,0 +1,8 @@
+%{
+# 
+slice                       : tinyint                       #
```
Comment (ecobost) [inline matlab/schemas/+shared/SliceTest.m]: I think I created this one and forgot to delete, we shouldn't have it here

File: matlab/schemas/+vis/TrialCopy171002.m [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+trial_idx                   : int                           # trial index within sessions
+---
+cond_idx                    : smallint unsigned             # condition index
+flip_times                  : mediumblob                    # (s) row array of flip times
+last_flip_count             : int unsigned                  # the last flip number in this trial
+trial_ts=CURRENT_TIMESTAMP  : timestamp                     # automatic
+%}
+
+
+classdef TrialCopy171002 < dj.Manual
```
Comment (dimitri-yatsenko) [inline matlab/schemas/+vis/TrialCopy171002.m]: This is just a backup table.  We don't need it.

File: matlab/schemas/+map/MaskCoordinates.m [hunk 4]
```diff
@@ -0,0 +1,22 @@
+%{
+# mask center of mass of a segmented cell
+-> map.ScanCoordinates
+-> `pipeline_preprocess`.`_extract_raw__galvo_r_o_i`
```
Comment (dimitri-yatsenko) [inline matlab/schemas/+map/MaskCoordinates.m]: Class not found

File: matlab/schemas/+map/LayerMembership.m [hunk 5]
```diff
@@ -0,0 +1,21 @@
+%{
+# 
+-> `pipeline_preprocess`.`#slice`
```
Comment (dimitri-yatsenko) [inline matlab/schemas/+map/LayerMembership.m]: classes not found

**Pull request merged**